1 table.sql

```diff
@@ -0,0 +1,18 @@
+drop TABLE if EXISTS dbo.CleaningCompany
+go
+CREATE table dbo.CleaningCompany(
+        CustomerID int not null IDENTITY primary key,
```

Comment:
The PK should be the name of the table + Id. Either change the table name or the PK column name.